nf_core/pipeline-template/docs/usage.md

```diff
@@ -246,6 +246,14 @@ See the main [Nextflow documentation](https://www.nextflow.io/docs/latest/config
 
 If you have any questions or issues please send us a message on [Slack](https://nf-co.re/join/slack) on the [`#configs` channel](https://nfcore.slack.com/channels/configs).
 
+## Azure Resource Requests
+
+To be used with the `azurebatch` profile by specifying the `-profile azurebatch`.
+We recommend providing a compute `params.vm_type` of `Standard_E64_v3` VMs by default but these options can be changed if required.
```

Comment:
I think we should change this language: `Standard_E64_v3` as this is mostly meant for Sarek.

I would change it to
We recommend providing a compute `params.vm_type` of `Standard_D16_v3` VMs by default but these options can be changed if required. For a thorough list, please refer the [Azure VM Sizes](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes).